wgpu-hal/src/lib.rs

```diff
@@ -700,20 +827,36 @@ pub trait Queue: WasmNotSendSync {
     ///
     /// Valid usage:
     ///
-    /// - All of the [`CommandBuffer`][cb]s were created from
-    ///   [`CommandEncoder`][ce]s that are associated with this queue.
+    /// - All of the [`CommandBuffer`][cb]s must have been created from
+    ///   [`CommandEncoder`][ce]s that were constructed from the [`Device`][d]
+    ///   associated with this [`Queue`].
+    ///
+    /// - A given [`CommandBuffer`][cb] must only be submitted for execution
```

Comment:
suggestion: In line with my other suggestion to use `Each…`:

```suggestion
    /// - Each [`CommandBuffer`][cb] must only be submitted for execution
```

I know this is from older documentation, so I'm happy to iterate in my own PR, if you want.